app/src/main/java/com/example/androidtraining/networking/API.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.androidtraining.networking
+
+import com.example.androidtraining.models.LoginResponse
+import com.example.androidtraining.models.Product
+import com.example.androidtraining.models.UserInfo
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.Headers
+import retrofit2.http.POST
+
+interface API {
+    @POST(value = "/api/auth/local")
+    @Headers("No-Authentication: true")
```

Comment:
Is the header really necessary? It should work without it, too.

Reply:
removed